lib/common/tests/xml/pcmk__xe_copy_attrs_test.c

```diff
@@ -14,49 +14,51 @@
 #include <glib.h>
 
 static void
-target_is_NULL(void **state)
+null_target(void **state)
 {
-    xmlNode *test_xml_1 = pcmk__xe_create(NULL, "test_xml_1");
-    xmlNode *test_xml_2 = NULL;
+    // This test dumps core via CRM_CHECK()
+    xmlNode *src = pcmk__xe_create(NULL, "test");
+    xmlNode *target = NULL;
 
-    pcmk__xe_set_props(test_xml_1, "test_prop", "test_value", NULL);
+    crm_xml_add(src, "attr", "value");
+    pcmk__xe_copy_attrs(target, src, true, false);
 
-    pcmk__xe_copy_attrs(test_xml_2, test_xml_1, true, false);
+    assert_ptr_equal(target, NULL);
```

Comment:
This doesn't really test anything unfortunately. The call couldn't reassign target simply per C. Best approach would be to have `pcmk__xe_copy_attrs()` return EINVAL and test that.

Reply:
You're right. The previous version included the same test and I didn't notice while cleaning up.